src/containers/SingleSignOn/IdleTimeoutHandler.jsx

```diff
@@ -43,7 +43,7 @@ export default class IdleTimeoutHandler extends PureComponent {
     if (this.props.router) {
       this.props.router.setRouteLeaveHook(this._getCurrentRoute(), null);
     }
-    userManager.forceSignoutRedirect();
+    createUserManager().forceSignoutRedirect();
```

Comment:
Again, if this component was unaware of the existence of the user manager, it might be better.